packages/@aws-cdk/aws-elasticloadbalancingv2/lib/nlb/network-target-group.ts

```diff
@@ -106,8 +106,8 @@ export class NetworkTargetGroup extends TargetGroupBase implements INetworkTarge
     if (healthCheck.path) {
       ret.push('Health check paths are not supported for Network Load Balancer health checks');
     }
-    if (healthCheck.protocol && healthCheck.protocol !== Protocol.TCP && healthCheck.protocol !== Protocol.TLS) {
-      ret.push(`Health check protocol '${healthCheck.protocol}' is not supported. Must be one of [TCP, TLS]`);
```

Comment:
This isn't the case according to the docs [the docs](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/target-group-health-checks.html):

> HealthCheckProtocol
> The possible protocols are HTTP, HTTPS, and TCP